routes/stats.js

```diff
@@ -88,17 +96,72 @@ statsRouter.get('/month', async (req, res) => {
     const response = await pool.query(
       `SELECT SUM(pounds) AS pound_sum, SUM(ounces) AS ounces_sum FROM event_data INNER JOIN events on event_data.event_id=events.id WHERE events.date BETWEEN CURRENT_DATE - INTERVAL '1 month' AND CURRENT_DATE`,
     );
-    const pounds = parseFloat(response.rows[0].pound_sum);
-    const ounces = parseFloat(response.rows[0].ounces_sum);
+    const pounds = response.rows[0].pounds_sum != null ? parseFloat(response.rows[0].pound_sum) : 0;
+    const ounces = response.rows[0].ounces_sum != null ? parseFloat(response.rows[0].ounces_sum) : 0;
     const total = pounds + ounces / 16;
 
     const responseTwo = await pool.query(
-      `SELECT SUM(pounds) AS pound_sum, SUM(ounces) AS ounces_sum FROM event_data INNER JOIN events on event_data.event_id=events.id WHERE events.date BETWEEN CURRENT_DATE - INTERVAL '3 months' AND CURRENT_DATE - INTERVAL '1 month' `,
+      `SELECT SUM(pounds) AS pound_sum, SUM(ounces) AS ounces_sum FROM event_data INNER JOIN events on event_data.event_id=events.id WHERE events.date BETWEEN CURRENT_DATE - INTERVAL '2 months' AND CURRENT_DATE - INTERVAL '1 month' `,
     );
-    const poundsTwo = parseFloat(responseTwo.rows[0].pound_sum);
-    const ouncesTwo = parseFloat(responseTwo.rows[0].ounces_sum);
+    console.log(responseTwo)
+    const poundsTwo = responseTwo.rows[0].pound_sum != null ? parseFloat(responseTwo.rows[0].pound_sum) : 0;
+    const ouncesTwo = responseTwo.rows[0].ounces_sum != null ? parseFloat(responseTwo.rows[0].ounces_sum) : 0;
     const totalTwo = poundsTwo + ouncesTwo / 16;
-    const rate = ((totalTwo - total) / total).toFixed(2);
+    const rate = (totalTwo != 0 ? ((total - totalTwo) / totalTwo) : total).toFixed(1);
```

Comment:
🚫 **[eslint]** <[eqeqeq](https://eslint.org/docs/latest/rules/eqeqeq)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected '!==' and instead saw '!='.